lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -408,6 +424,35 @@
       name: customimage001
       resource_group: Testing
 
+- name: Create VM with spcified OS disk size
+  azure_rm_virtualmachine:
+    resource_group: Testing
+    name: big-os-disk
+    admin_username: chouseknecht
+    admin_password: <your password here>
+    os_disk_size: 512
+    image:
+      offer: CentOS
+      publisher: OpenLogic
+      sku: '7.1'
+      version: latest
+
+- name: Create VM with OS and Plan, accepting the terms
+  azure_rm_virtualmachine:
+    resource_group: Testing
+    name: f5-nva
+    admin_username: chouseknecht
+    admin_password: <your password here>
+    image:
+      publisher: f5-networks
```

Comment:
need a free/cheap thing here, or make this task/arg conditional with `omit`

Reply:
I found what I think is the cheapest, mostly that it is $0.001/hr and will run on a `Standard_A0` VM.

Reply:
That's def cheap enough :)

Reply:
@nitzmahone can you please review your request for changes